test-common/test-common-api/src/main/java/org/apache/kafka/common/test/api/RaftClusterInvocationContext.java

```diff
@@ -222,6 +228,24 @@ public void waitForReadyBrokers() throws InterruptedException {
             }
         }
 
+        @Override
+        public int getLeaderBrokerId(TopicPartition topicPartition) throws ExecutionException, InterruptedException {
+            var leaderBrokerId = -1;
+            try (var admin = admin()) {
+                DescribeTopicsResult result = admin.describeTopics(List.of(topicPartition.topic()));
```

Comment:
Could you please consider using lambda to implement it? Also, this implement is using client APIs only so it can be moved to interface to be the default implementation

Reply:
@chia7712  Yeah, lambda definitely make the code more clearer.

 And I have moved the method to the interface also, PTAL